scripts/r.drain/r.drain.py

```diff
@@ -141,10 +140,11 @@ def main():
     atexit.register(cleanup)
 
     if not dirmap:
+        valmap_name = valmap.split("@")[0]
         # get directions with r.fill.dir, without sink filling
-        dirmap = "%s_tmp_%d" % (valmap, os.getpid())
-        fill_map = "%s_fill_%d" % (valmap, os.getpid())
-        area_map = "%s_area_%d" % (valmap, os.getpid())
+        dirmap = grass.append_node_pid(f"{valmap_name}_tmp")
```

Comment:
The dir map seems to be older code than fill and area maps, but otherwise, they are same, so just use `_dir` instead of `_tmp`.